erlang compile error

Wall -Wstrict-prototypes -Wmissing-prototypes -Wdeclaration-after-statement -DUSE_THREADS -D_THREAD_SAFE -D_REENTRANT -DPOSIX_THREADS -D_POSIX_THREAD_SAFE_FUNCTIONS -DERLANG_INTEGRATION -o /home/main/ar71xx/build_dir/target-mips_34kc_musl-1.1.10/otp_src_17.5/erts/emulator/pcre/obj/mips-openwrt-linux-gnu/opt/pcre_exec.o /home/main/ar71xx/build_dir/target-mips_34kc_musl-1.1.10/otp_src_17.5/erts/emulator/pcre/pcre_exec.c
{standard input}: Assembler messages:
{standard input}:1438: Error: operand value out of range for instruction
make[7]: *** [/home/main/ar71xx/build_dir/target-mips_34kc_musl-1.1.10/otp_src_17.5/erts/emulator/pcre/obj/mips-openwrt-linux-gnu/opt/pcre_exec.o] Error 1
make[7]: Leaving directory `/home/main/ar71xx/build_dir/target-mips_34kc_musl-1.1.10/otp_src_17.5/erts/emulator'
make[6]: *** [opt] Error 2
make[6]: Leaving directory `/home/main/ar71xx/build_dir/target-mips_34kc_musl-1.1.10/otp_src_17.5/erts/emulator'
make[5]: *** [opt] Error 2
make[5]: Leaving directory `/home/main/ar71xx/build_dir/target-mips_34kc_musl-1.1.10/otp_src_17.5/erts'
make[4]: *** [emulator] Error 2
make[4]: Leaving directory `/home/main/ar71xx/build_dir/target-mips_34kc_musl-1.1.10/otp_src_17.5'
make[3]: *** [noboot] Error 2
make[3]: Leaving directory `/home/main/ar71xx/build_dir/target-mips_34kc_musl-1.1.10/otp_src_17.5'
make[2]: *** [/home/main/ar71xx/build_dir/target-mips_34kc_musl-1.1.10/otp_src_17.5/.built] Error 2
make[2]: Leaving directory `/home/main/ar71xx/feeds/packages/lang/erlang'
make[1]: *** [package/feeds/packages/erlang/compile] Error 2
make[1]: Leaving directory `/home/main/ar71xx'
make: *** [package/erlang/compile] Error 2

--- a/lang/erlang/Makefile
+++ b/lang/erlang/Makefile
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=erlang
 PKG_VERSION:=17.5
-PKG_RELEASE:=2
+PKG_RELEASE:=3
 
 PKG_SOURCE:=otp_src_$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:= http://www.erlang.org/download/ \
@@ -21,6 +21,7 @@ PKG_LICENSE_FILES:=EPLICENCE
 PKG_MAINTAINER:=Nicolas Thill <nico@openwrt.org>
 
 PKG_BUILD_DEPENDS:=erlang/host openssl
+PKG_USE_MIPS16:=0
 
 HOST_BUILD_DIR:=$(BUILD_DIR_HOST)/otp_src_$(PKG_VERSION)
 PKG_BUILD_DIR:=$(BUILD_DIR)/otp_src_$(PKG_VERSION)

